content/en/news/releases/1.9.x/announcing-1.9/_index.md

```diff
@@ -0,0 +1,94 @@
+---
+title: Announcing Istio 1.9
+linktitle: 1.9
+subtitle: Major Update
+description: Istio 1.9 release announcement.
+publishdate: 2021-02-09
+release: 1.9.0
+skip_list: true
+aliases:
+    - /news/announcing-1.9
+    - /news/announcing-1.9.0
+---
+
+We are pleased to announce the release of Istio 1.9! Officially supported versions of Kubernetes releases are 1.17-1.20
+for this Istio release.
+
+{{< relnote >}}
+
+This is the first Istio release for 2021.  We would like to thank the entire Istio community, and especially the release
+managers Shamsher Ansari (RedHat), Steven Landlow (Google) and Jacob Delgado (Aspen Mesh) for helping to get Istio 1.9.0
+published.
+
+Given the work done by the UX Group, notably Mitch Connors (Google) and Christian Posta (Solo.io), and led by the
+direction of the [Istio TOC](https://github.com/istio/community/blob/master/TECH-OVERSIGHT-COMMITTEE.md), Istio 1.9's
+core focus was to improve Day 2 operations. These include stabilizing installation, upgrades, core functionality and
+features, based on the criteria on the [Feature Status](https://istio.io/latest/about/feature-stages/) page. Thank you
+to everyone that participated in the survey in helping to make Istio 1.9 be our most successful release to date.
+
+More information will be published in the future detailing the Istio community's plans for 2021 in an upcoming blog.
+
+Here are some highlights for this release:
+
+## Feature promotion
+
+### Virtual machine installation promoted to Beta
+
+With many users needing to expand their service mesh to include virtual machines from existing systems, managing the
+complexity of installation is paramount to user success. As such, virtual Machine support has continued to be a high
+priority for the Istio Community. Since Istio 1.6, the community has made major improvements each release for VM
+Support. As of Istio 1.9, [Virtual Mach Installation](/docs/setup/install/virtual-machine/) has been promoted to Beta.
+
+VM support will continue to be a high priority for us in 2021, and you can expect us to see more feature
+graduation in upcoming releases.
+
+### External Istiod promoted to Beta
+
+As with VMs, management of external clusters is vital as Mesh Admins increasingly need to expand their service meshes
+to include remote clusters for many applications (e.g. global distribution, high availability). By promoting this
+feature to Beta we recognize this as a must for adoption and to promote widespread-usage.
+
+## service-apis promoted to Alpha
```

Comment:
cc @howardjohn Please take a closer look at this section. Improvements using the "insert a suggestion" are welcome.